### ISSUE TITLE
CRITICAL: Cannot instantiate abstract class Oro\Bundle\UIBundle\Twig\Template

### ISSUE THREAD

**Issue by dainiusgliebus** [oroinc/platform#890]
Version 3.1.0

php.CRITICAL: Cannot instantiate abstract class Oro\Bundle\UIBundle\Twig\Template {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Cannot instantiate abstract class Oro\\Bundle\\UIBundle\\Twig\\Template at /oro/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Caster/ExceptionCaster.php:215)"} 

request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Cannot instantiate abstract class Oro\Bundle\UIBundle\Twig\Template" at /oro/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Caster/ExceptionCaster.php line 215 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Cannot instantiate abstract class Oro\\Bundle\\UIBundle\\Twig\\Template at /oro/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Caster/ExceptionCaster.php:215)"}

**Comment by anyt** [comment]
HI @dainiusgliebus,
is it clean installation or result of upgrade?

**Comment by dainiusgliebus** [comment]
@anyt clean installation as well tried to install few times again and again, but still same issue

**Comment by anyt** [comment]
please share the command, how you checked out the repository, or the link, where did you get it

**Comment by vtsykun** [comment]
I have the same problem. I think it's related to https://github.com/symfony/symfony/issues/27921 bug.

**Comment by anyt** [comment]
You could remove the configuration for twig.base_template_class from `config/config_dev.yml`
```
twig:
    base_template_class: Oro\Bundle\UIBundle\Twig\Template
```
Probably you are not using this feauture anyway.

**Comment by anyt** [comment]
This abstract template adds comments to the page source code in dev environemt in order to easier find original twig templates. But when you are using twig-inspector, it is useless. I will create an improvement to delete this configuration.
Thanks for the report.

**Comment by anyt** [comment]
The fix merged, will be available with the next 3.1 patch release.
